.travis.yml

```diff
@@ -6,32 +6,38 @@ env:
     else git log -1 --skip 1 --pretty=format:%ct; fi)
 sudo: false
 language: ruby
-cache: bundler
+cache:
+  bundler: true
+  yarn: true
 branches:
   only:
-  - master
-  - /^v\d+\.\d+\.\d+$/
+    - master
+    - /^v\d+\.\d+\.\d+$/
 services:
-- postgresql
-- elasticsearch
+  - postgresql
+  - elasticsearch
 addons:
   chrome: stable
 before_install:
-- gem update --system
+  - gem update --system
+install:
+  - bundle install --path vendor/bundle
+  - yarn
 before_script:
-- sleep 10
-- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
-  > ./cc-test-reporter
-- chmod +x ./cc-test-reporter
-- "./cc-test-reporter before-build"
+  - sleep 10
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
+    > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - "./cc-test-reporter before-build"
 script:
-- bundle exec brakeman
-- bundle exec rake db:create
-- bundle exec rake db:migrate
-- bundle exec rake spec
+<<<<<<< HEAD
```

Comment:
We don't want to include this.

How about we rebase this branch and rework the commits now that we've figured out an approach that works?

Reply:
Thanks for spotting - this is the n'th rebase and I must have missed it.

Reply:
You're right that there is a lot on this branch, though. I'll try and get them into a separate PR.